src/commands/tui/snapshots.rs

```diff
@@ -552,6 +555,17 @@ impl<'a, P: ProgressBars, S: IndexedFull> Snapshots<'a, P, S> {
         self.get_snap_entity(|snap| snap.description.clone().unwrap_or_default())
     }
 
+    pub fn get_filter(&self) -> Result<String> {
+        Ok(toml::to_string_pretty(&self.filter)?)
+    }
+
+    pub fn set_filter(&mut self, filter: String) {
+        if let Ok(filter) = toml::from_str::<SnapshotFilter>(&filter) {
```

Comment:
is there something to warn of an error in the TUI interface ?

Reply:
Let's merge it and postpone those warnings...